cms/templates/js/video-thumbnail.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="thumbnail-wrapper <%- action === 'upload' ? 'upload' : '' %>" tabindex="-1">
+    <img src="<%- thumbnailURL %>" alt="<%- imageAltText %>">
+    <div class="thumbnail-overlay">
+        <input id="thumb-<%- videoId %>" class="upload-image-input" type="file" name="file" accept=".bmp, .jpg, .jpeg, .png, .gif"/>
+        <label for="thumb-<%- videoId %>" class="thumbnail-action">
+            <span class="action-icon" aria-hidden="true"><%- actionInfo.icon %></span>
+            <span class="action-text-sr sr"></span>
+            <span class="action-text"><%- actionInfo.text %></span>
+        </label>
+
+        <span class="requirements-text-sr sr">
+            <%- edx.StringUtils.interpolate(
+                gettext("Recommended image resolution is {imageResolution}, maximum image file size should be {maxFileSize} and format must be one of {supportedImageFormats}."),
```

Comment:
Do we recommend a particular image resolution, or do we require it? Also, can these three requirements be on separate lines? For example:

The image resolution must be {imageResolution}. 
Image files must be {maxFileSize} or smaller. 
The file type must be .gif, .jpg, or .png.